pkg/helpers/osidentity/osidentity.go

```diff
@@ -11,6 +13,16 @@ func NewMacOSForTest() *Identity {
 	return &Identity{"darwin", ""}
 }
 
+// NewFromRuntime returns new identity from runtime platform
+func NewFromRuntime() *Identity {
```

Comment:
I wonder if we should just name that `New()` since there is only one obvious way to create this.
(I know there is other examples of this pattern)

Reply:
After reading the full change: I think there is no point having those two constructors since we will always call `Detect()` to instantiate an `os.Identity`.

Reply:
yeah you're right, it's useless to add `New....` since anyway we would always call `Detect`